src/locales/hi-IN.json

```diff
@@ -170,7 +170,7 @@
     "zoomToFit": ""
   },
   "encrypted": {
-    "tooltip": ""
+    "tooltip": "आपके चित्र अंत-से-अंत एन्क्रिप्टेड हैं, इसलिए एक्सक्लूसिव्रॉव के सर्वर उन्हें कभी नहीं देखेंगे।"
```

Comment:
As for this.. the translation should happen here: https://crowdin.com/project/excalidraw/hi# and it's only at 7% right now.

Reply:
```suggestion
    "tooltip": ""
```